maintainers/maintainer-list.nix

```diff
@@ -11052,4 +11052,10 @@
     github = "zupo";
     githubId = 311580;
   };
+  xdhampus = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.